packages/scalecube-microservice/src/helpers/utils.ts

```diff
@@ -18,10 +18,4 @@ export const throwErrorFromServiceCall = ({
   return asyncModel === ASYNC_MODEL_TYPES.REQUEST_RESPONSE ? Promise.reject(error) : throwError(error);
 };
 
-/* tslint:disable */
-// https://stackoverflow.com/questions/105034/create-guid-uuid-in-javascript
-export const uuidv4 = () =>
-  ('' + 1e7 + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, (c: any) =>
-    (c ^ (crypto.getRandomValues(new Uint8Array(1))[0] & (15 >> (c / 4)))).toString(16)
-  );
-/* tslint:enable */
+export const isNodeEnv = () => typeof window === 'undefined';
```

Comment:
seem obsolete and duplication

Reply:
@idanilt This is a helper in a different package, I thought some helpers, if they are required in different packages

Reply:
@idanilt Or are you suggesting to create a package for common helpers?